app/views/welcome/index.html.erb

```diff
@@ -1,3 +1,8 @@
+<%if user_signed_in?%>
```

Comment:
- [ ] Please use scoping or ordering SQL statements in the transactions controller `index` restful method to sort the transactions from latest to oldest as required.
![Screenshot (1)](https://user-images.githubusercontent.com/47625407/221093660-f2ad92b9-52fc-4ce8-9814-94a541642475.png)
It was expected to be:
```
Charge2 ...
Charge1 ...
```

_Please notice that the corresponding file for this requirement isn't loaded in your PR, so I had to leave the comment in a random file._

Reply:
fixed

Reply:
This issue has been fixed